store.css

```diff
@@ -0,0 +1,199 @@
+/*STYLING FOR HEADER*/
+
+#karma-logo {
+  width: 25px;
+  margin-left: 50px;
+  margin-top: 50px;
+  margin-bottom: 20px;
+}
+
+nav {
+  float: right;
+  margin-top: 50px;
+  margin-right: 50px;
+}
+
+nav a {
+  margin-right: 40px;
+  text-decoration: none;
+  color: gray;
+}
+
+#karma-meet {
+  font-weight: bold;
+}
+a:hover {
+  color: #c05326;
+}
+
+/*STYLING FOR BODY*/
+
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*main {
+  background-image: url(level-2/store-image_by-andrew-neel-unsplash.jpg);
+  position: absolute;
+}*/
+
+h1 {
+  color: #de6e49;
+  font-size: 50px;
+  font-weight: lighter;
+  width: 500px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.image {
+  width: 50%;
+}
+
+.image img {
+  width: 100%;
+  height: 100%;
+}
+
+section {
+  display: flex;
+  justify-content: space-between;
+  border-top: 1px solid #ebebeb;
+  width: 100%;
+  gap: 50px;
+}
+
+.title-form {
+  flex-direction: column;
```

Comment:
this is missing: 
display: flex
because flex-direction: column on its own will not do anything to a container that has not been defined as a flexbox